src/adr/articles/0013-aws-waf.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: layouts/adr.njk
+title: "AWS WAF"
+weight: 1
+date: 2022-03-22
+review_in: 12 months
+tags:  
+    - adr
+    - observability
+    - tools
+    - security_threats
+    - observability
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0013"
+---
+
+## Context
+
+We want to ensure our services are protected to a high standard via a Web Application Firewall.
```

Comment:
Maybe some more detail on the kinds things we are protected against and the rulesets and why we use them? 

We currently use the: 

- CommonRulesSet
- KnownBadInputs
- PHPRulesSet

On the PHP apps, so maybe something abstracted to say rulesets that cover common attacks like SQL injection, XSS and known language vulnerabilities.